WooCommerce/Classes/POS/ViewModels/PointOfSaleDashboardViewModel.swift

```diff
@@ -149,6 +193,52 @@ private extension PointOfSaleDashboardViewModel {
     }
 }
 
+private extension PointOfSaleDashboardViewModel {
+    private var shouldSyncOrder: Bool {
+        return orderStage == .finalizing
+    }
+
+    private func createOrUpdateOrder() async {
```

Comment:
I moved this logic into this method since we are triggering order creation/updates a bit more manual now so we do not need to do it on every cart update